lib/bpk-theming/src/create-with-theme.js

```diff
@@ -30,6 +31,14 @@ const createWithTheme = () => {
 
   return Component => {
     class WithTheme extends React.Component {
+      static propTypes = {
+        forwardedRef: PropTypes.func,
```

Comment:
Ref is not always a function, the correct type is 
```js
PropTypes.oneOfType([
    // Either a function
    PropTypes.func, 
    // Or the instance of a DOM native element (see the note about SSR)
    PropTypes.shape({ current: PropTypes.instanceOf(Element) })
])
```

Reply:
I copied this from another component for consistency. Will change them both to this format.

Reply:
Actually wouldn't this only apply to web? On RN, there is no built in `Element` right? And if it's an instance of a component, components are functions.

Reply:
I think element in this case is `React.Element` which is the same for both. The second version is what `useRef` or `React.createRef` returns.